scripts/test_balance_check.py

```
@@ -0,0 +1,86 @@
+                    print(f"[{function_name}] 💰 구매 가능 금액 (외화사용가능금액): ${available_cash:,.2f}")
+                    
+                    # 추가 디버깅 정보
+                    frcr_evlu_tota = output3.get("frcr_evlu_tota", "0")
+                    print(f"[{function_name}] 📊 외화평가총액: ${float(frcr_evlu_tota):,.2f}")
+                    
+                    # 테스트 결과
+                    print()
+                    print("=" * 80)
+                    print("✅ 테스트 결과")
+                    print("=" * 80)
+                    print(f"• API 호출 성공: ✅")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)